packages/streamline/src/components/cards/card-carousel/card-carousel.tsx

```diff
@@ -67,31 +67,37 @@ export const CardCarousel = (props: CardCarouselProps) => {
         style={StyleSheet.absoluteFillObject}
       />
       <Box
-        flexDirection="row-reverse"
+        flexDirection="row"
         justifyContent="space-between"
         alignItems="center"
       >
-        <ButtonIcon
-          testID={`${testID}-close-button`}
-          isLoading={isLoading || isResolving}
-          iconName="Cross"
-          accessibilityLabel="close"
-          appearance="light"
-          onPress={onClose}
-          withContainer
-        />
+        <Box>
+          {isSmall && props.iconName ? (
+            <Icon iconName={props.iconName} color="PURE_WHITE_1000" />
+          ) : null}
 
-        {size === 'large' && props.tag ? (
-          <Tag
-            text={props.tag.text}
-            iconName={props.tag.iconName}
-            appearance="dark"
-          />
-        ) : null}
+          {!isSmall && props.tag ? (
+            <Tag
+              text={props.tag.text}
+              iconName={props.tag.iconName}
+              appearance="dark"
+            />
+          ) : null}
+        </Box>
 
-        {size === 'small' && props.iconName ? (
-          <Icon iconName={props.iconName} color="PURE_WHITE_1000" />
-        ) : null}
+        <Box>
```

Comment:
Is the `Box` necessary?

Reply:
The header is composed of 2 elements : The tag and the ButtonIcon (onClose), and are handle with a `space-between`. 
Problem : `Tag` is optional . In the case where there's no "Tag" but a "onClose", the ButtonIcon will be on the top-left-side instead of the top-right-side.